contracts/OctobayGovToken.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+import './OctobayStorage.sol';
+
+contract OctobayGovToken is OctobayStorage, ERC20 {
+
```

Comment:
I think here we'll need properties like `minProposalSuccessQuorum`, `minProposalRejectQuorum` and `minShareForProposing`

Reply:
Given what we discussed now, I guess these aren't required now?

Reply:
true